### ISSUE TITLE
Excuse me.

### ISSUE THREAD

**Issue by spiside** [FrankPortman/dotfiles#1]
I can't seem to find any code in this repo. Is this in error?

**Comment by FrankPortman** [comment]
@spiside yes this is a huge error pls give me some time to git stash and git pull and git reorganize

**Comment by spiside** [comment]
OOOOO! Now I see them. Thanks for updating <3
